Tests/fixtures/podcasting.yml

```diff
@@ -0,0 +1,9536 @@
+  os:
+    name: Android
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Podkicker
+    version: "3.7.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podkicker Pro/3.1.19.RC-GP-Pro(2028)'
+  os:
+    name: Android
+    version: ""
+    platform: ""
+  client:
+    type: mobile app
+    name: Podkicker Pro
+    version: "3.1.19"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (Mobile; LYF/F271i/LYF_F271i-000-01-20-101019; Android; rv:48.0) Gecko/48.0 Firefox/48.0 KAIOS/2.5 PodLP/1.3.2.0'
+  os:
+    name: KaiOS
+    version: "2.5"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodLP
+    version: "1.3.2.0"
+  device:
+    type: smartphone
+    brand: LYF
+    model: 'F271i'
+  os_family: Firefox OS
+  browser_family: Unknown
+
+-
+  user_agent: 'Mozilla/5.0 (Mobile; Nokia 2720 V Flip; rv:48.0; CAEN) Gecko/48.0 Firefox/48.0 KAIOS/2.5.4 PodLP/2.2.1'
+  os:
+    name: KaiOS
+    version: "2.5.4"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodLP
+    version: "2.2.1"
+  device:
+    type: smartphone
+    brand: Nokia
+    model: '2720'
+  os_family: Firefox OS
+  browser_family: Unknown
+
+-
+  user_agent: 'Podme android app/5.1 (Linux;Android 9) ExoPlayerLib/2.11.5'
+  os:
+    name: Android
+    version: "9"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodMe
+    version: "5.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podme android app/6.10.0 (Linux;Android 13) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodMe
+    version: "6.10.0"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podme android app/6.7.4 (Linux;Android 12) ExoPlayerLib/2.18.2'
+  os:
+    name: Android
+    version: "12"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodMe
+    version: "6.7.4"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'Podme android app/6.5.1 (Linux;Android 13) ExoPlayerLib/2.17.1'
+  os:
+    name: Android
+    version: "13"
+    platform: ""
+  client:
+    type: mobile app
+    name: PodMe
+    version: "6.5.1"
+  device:
+    type: ""
+    brand: ""
+    model: ''
+  os_family: Android
+  browser_family: Unknown
+
+-
+  user_agent: 'PodMe/10289 CFNetwork/1240.0.4 Darwin/20.6.0'
+  os:
+    name: iOS
+    version: "14.7"
+    platform: ""
+  client: null
+  device:
+    type: ""
+    brand: Apple
+    model: ''
+  os_family: iOS
+  browser_family: Unknown
+
+-
+  user_agent: 'PodMN/iOS 1.1.11 (iPhone/16.5.1)'
+  os:
+    name: iOS
+    version: "1.1.11"
```

Comment:
wrong version 
version: "16.5.1"

Reply:
This required a case in the IOS detection to avoid.